docs/site/Server.md

```diff
@@ -58,6 +58,52 @@ export class HelloWorldApp extends RestApplication {
 
 ## Configuration
 
+### Enable HTTPS
+
+Enabling HTTPS for the LoopBack REST server is just a matter of specifying the
+protocol as `https` and specifying the crendential files. You can either use a
+key-certificate or pfx-passphrase combo.
```

Comment:
```markdown
You can configure the server-side credentials by providing either a PFX/PKCS12 data,
or a pair of a private key and a cert chain. Please refer to Node.js documentation for
[https.createServer()](https://nodejs.org/api/https.html#https_https_createserver_options_requestlistener)
for the description of all available options.
```

Note that `passphrase` can be used for both PFX and key/cert mode. See https://nodejs.org/api/tls.html#tls_tls_createsecurecontext_options
> passphrase `<string>` Optional shared passphrase used for a single private key and/or a PFX.